File: digitalpassbook2/app/src/main/java/com/example/digitalpassbook/Club.java
package com.example.digitalpassbook;
import com.google.gson.annotations.SerializedName;

public class Club {
    @SerializedName("id")
    int id;

    @SerializedName("clubname")
    String clubname;

    public Club (int id, String clubname) {
        this.id = id;
        this.clubname = clubname;
    }
    public Club (String clubname) {
        this.clubname = clubname;
    }
}

File: digitalpassbook2/app/src/main/java/com/example/digitalpassbook2/Club.kt
package com.example.digitalpassbook2

import com.google.gson.annotations.SerializedName

class Club {
    @SerializedName("id")
    var id = 0

    @SerializedName("name")
    var name: String

    constructor(id: Int, name: String) {
        this.id = id
        this.name = name
    }

    constructor(name: String) {
        this.name = name
    }
}